brightwind/load/station.py

```diff
@@ -972,7 +972,7 @@ def get_table(self, detailed=False, wind_speeds=False, wind_directions=False, ca
             order_index = dict(zip(MEAS_TYPE_ORDER, range(len(MEAS_TYPE_ORDER))))
```

Comment:
can you update the doc string of the get_table function? as you can see on screenshot below this is not reporting the correct description for most of the parameters
<img width="526" alt="image" src="https://github.com/user-attachments/assets/142826a6-631a-4336-b154-7a1681fafc67" />